daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -176,7 +176,7 @@ func (l *logStream) Log(msg *logger.Message) error {
 	defer l.lock.RUnlock()
 	if !l.closed {
 		// buffer up the data, making sure to copy the Line data
-		l.messages <- logger.CopyMessage(msg)
+		l.messages <- msg
```

Comment:
The comment says the data is being copied, but it's no longer being copied.

Is it okay to send `msg` over the channel without copying it? A caller might call `Log` and then modify `msg` afterwards.